tests/scripts/all.sh

```diff
@@ -2610,14 +2610,11 @@ component_test_no_psa_crypto_full_cmake_asan() {
     msg "test: ssl-opt.sh (full minus PSA crypto)"
     tests/ssl-opt.sh
 
-    msg "test: compat.sh default (full minus PSA crypto)"
-    tests/compat.sh
-
-    msg "test: compat.sh NULL (full minus PSA crypto)"
+    msg "test: compat.sh: NULL (full minus PSA crypto)"
     tests/compat.sh -f 'NULL'
```

Comment:
We aren't actually running any tests here!

```suggestion
    tests/compat.sh -e '^$' -f 'NULL'
```

Reply:
Ouch, embarrassing. Thanks for noticing.